internal/testutil/apps/constant.go

```diff
@@ -54,7 +54,7 @@ const (
 	DefaultRedisCompSpecName      = "redis-rsts"
 	DefaultRedisImageName         = "redis:7.0.5"
 	DefaultRedisContainerName     = "redis"
-	DefaultRedisInitContainerName = "redis-init-container"
+	DefaultRedisInitContainerName = "redis-Init-container"
```

Comment:
Don't touch.

Reply:
revert in https://github.com/apecloud/kubeblocks/pull/5306